### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by joseleoc**
auth0 persistence
fix some visual issues

### CODE REVIEW

**Review by lagoonStudios — APPROVED** [review]

File: hooks/Auth.tsx [hunk 1]
```diff
@@ -25,8 +25,6 @@ export function useAuth0Config() {
     setLoaded(false);
     const envDomain = process.env.EXPO_PUBLIC_AUTH0_DOMAIN;
     const envClientId = process.env.EXPO_PUBLIC_AUTH0_CLIENT_ID;
-    // const envDomain = "dev-ghex3xcv4jb2pti7.us.auth0.com";
```
Comment (Raidenmystery) [inline hooks/Auth.tsx:28]: @joseleoc delete unused comments

File: hooks/Auth.tsx [hunk 2]
```diff
@@ -65,12 +63,15 @@ export default function useAuth() {
         realm: CONNECTION,
         audience: AUDIENCE,
       });
-
-      auth.credentialsManager.saveCredentials(credentials);
-      // const credentials = await auth?.credentialsManager.getCredentials();
```
Comment (Raidenmystery) [inline hooks/Auth.tsx:70]: @joseleoc delete unused comments

File: hooks/Auth.tsx [hunk 3]
```diff
@@ -65,12 +63,15 @@ export default function useAuth() {
...
-      // console.log({ credentials });
-
+      // console.log("Login credentials:", { credentials });
+      console.log({ credentials });
+      await auth.credentialsManager.saveCredentials({
+        ...credentials,
+        idToken: credentials.accessToken,
+      });
+      // auth.credentialsManager.getCredentials().then((creds) => {
+      //   console.log("checkCredentials from manager", { creds });
```
Comment (Raidenmystery) [inline hooks/Auth.tsx:73]: @joseleoc delete unused comments

File: providers/Auth.tsx [hunk 4]
```diff
@@ -29,26 +28,30 @@ export default function AuthProvider({
     return auth?.credentialsManager.clearCredentials();
   };
 
-  const checkCredentials = useCallback(() => {
-    console.log({ test: "test1" });
-    auth?.credentialsManager.getCredentials().then((creds) => {
-      console.log("checkCredentials", { creds });
-    });
-    // const logged = await auth?.credentialsManager.hasValidCredentials();
```
Comment (Raidenmystery) [inline providers/Auth.tsx:37]: @joseleoc delete unused comments

File: providers/Auth.tsx [hunk 5]
```diff
@@ -29,26 +28,30 @@ export default function AuthProvider({
     return auth?.credentialsManager.clearCredentials();
   };
 
-  const checkCredentials = useCallback(() => {
-    console.log({ test: "test1" });
```
Comment (Raidenmystery) [inline providers/Auth.tsx:33]: @joseleoc delete unused logs

File: providers/Auth.tsx [hunk 6]
```diff
@@ -29,26 +28,30 @@ export default function AuthProvider({
...
   useEffect(() => {
     checkCredentials();
-  }, [checkCredentials]);
+  }, [token]);
   // --- END: Side effects -----------------------------------------------------
 
   return (
-    <Auth0Provider domain={domain} clientId={clientId}>
+    <Auth0Provider
+      domain={"dev-ghex3xcv4jb2pti7.us.auth0.com"}
```
Comment (Raidenmystery) [inline providers/Auth.tsx:52]: @joseleoc Add env variable here instead of mock value

**Review by Raidenmystery — APPROVED** [review]

**Pull request merged**